sdks/python/apache_beam/yaml/yaml_transform.py

```diff
@@ -771,6 +771,9 @@ def preprocess_windowing(spec):
         'transforms': [modified_spec] + windowing_transforms,
         'input': spec['input'],
         'output': modified_spec['__uuid__'],
+        'config': {
+            'error_handling': spec.get('config').get('error_handling', None)
```

Comment:
Nit, let's use `spec['config'].get('error_handling', None)` for consistency (unless we're worried the spec might not have a config, in which case we'd want `spec.get('config', {})` (which I think we'd need as it happens before the `ensure_config` phase).

Also, should it be `get('error_handling', {})` rather than `get('error_handling', None)`

Reply:
Agreed with both